actions/lib/formatters.py

```diff
@@ -14,6 +15,45 @@
 ]
 
 
+def branch_protection_to_dict(branch_protection):
+    result = {}
+    for attr in branch_protection_attributes:
+
+        if hasattr(branch_protection, attr):
+            if attr == 'required_status_checks':
+                req_status_checks = branch_protection.required_status_checks
+                if req_status_checks:
+                    result[attr] = {'contexts': req_status_checks.contexts,
+                                    'strict': req_status_checks.strict}
+                else:
+                    result[attr] = None
+                continue
+            if attr == 'required_pull_request_reviews':
+                req_pr_reviews = branch_protection.required_pull_request_reviews
+                if req_pr_reviews:
+                    result[attr] = {}
+                    for attr2 in required_pull_request_reviews_attributes:
+                        if attr2 == 'dismissal_users':
+                            users = []
+                            for user in req_pr_reviews.dismissal_users:
+                                users.append(user.login)
+                            result[attr][attr2] = users
+                            continue
+                        if attr2 == 'dismissal_tems':
```

Comment:
same stuff in here, you can avoid the `continue` by just doing an `elif` and `else`